openmeetings-web/src/main/java/org/apache/openmeetings/web/room/wb/raw-wb-area.js

```diff
@@ -131,6 +131,14 @@ var DrawWbArea = function() {
 		}
 		const link = li.find('a')
 			, wbId = link.data('wb-id');
+		// Apply right click menu only to the text item
+		li.find(".nav-link").find("span").first().contextmenu(
+			function(e) {
```

Comment:
There is not need to check for PRESENTER here cause this code method would already return/terming early in case role !== PRESENTER.

Reply:
Not sure, maybe it was added to prevent some hacks